arkouda/series.py

```diff
@@ -111,7 +116,7 @@ class Series:
     def __init__(
         self,
         data: Union[Tuple, List, groupable_element_type],
-        index: Optional[Union[pdarray, Strings, Tuple, List, Index]] = None,
+        index: Optional[Union[pdarray, Strings, Tuple, List]] = None,
```

Comment:
This is the line that was added in #1508 to enable `index` being an `Index`

I'm pretty sure what happened here is you had a merge conflict and you took your changes over the ones in `master`, but we def want the ones in `master` since this was part of a critical bug fix. So if you put this back, you won't need to touch `dataframe.py`

Reply:
Put back the code in dataframe the way it was.